src/components/past-events/past-events.jsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+import classes from './past-events.less';
+
+const DiamondAngle = '26deg';
```

Comment:
* why PascalCase is used for names of these variables?
* I think size and angle of diamonds should be properties